components/MnavBar.js

```diff
@@ -2,7 +2,44 @@ import React, { useState } from 'react';
 
 import Image from 'next/image';
 import Logo from '../public/logo.svg';
-import Button from '../components/form/Button';
+import Link from 'next/link';
+import { WhiteLogo, HamburgerMenu } from '../assets';
+
+const style = {
+  container: `relative top-1/4 w-full text-left pl-16 md:pl-32 mt-8`,
+  item: `text-3xl text-amcovad-white cursor-pointer  hover:amcovad-secondary-100`,
+  logo: `absolute top-8 left-8 md:left-20`,
+  menu: {
+    isMenuOpen: `h-full w-full `,
```

Comment:
`isMenuOpen` is highly ambiguous and doesn't flow well with the other values in that object. Lets use `open` to make it more explanatory.

```suggestion
    open: `h-full w-full `,
```